data/model/oci/tag.py

```diff
@@ -191,6 +191,7 @@ def get_most_recent_tag_lifetime_start(repository_ids):
     specified repository or None if none.
     """
     assert len(repository_ids) > 0 and None not in repository_ids
+    assert len(repository_ids) <= 100
```

Comment:
Interesting approach. It's impossible to know this requirement just from the function caller's perspective of the function signature. Is that a normal pattern?

Reply:
It is documented in the interface that calls this, but yeah, in Python there isn't really a good way to know.